OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -156,6 +156,7 @@
     <Compile Include="Primitives\MergedStream.cs" />
     <Compile Include="Primitives\SpatiallyPartitioned.cs" />
     <Compile Include="Primitives\ConcurrentCache.cs" />
+    <Compile Include="SelectableExts.cs" />
```

Comment:
Well, in theory... yes.
In practice, however, we have tons of exceptions for every rule. And this is tightly coupled with the `Selectable` trait.
Also I don't consider it to be an engine thing.
Anyway, let's hear a third opinion.